src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,52 @@
 import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movieData: {
+    title: string,
+    description?: string,
+    imgUrl: string,
+    imdbUrl: string,
+  }
+};
+
+export const MovieCard: React.FC<Props> = ({ movieData }) => {
+  const {
+    title, description = '', imgUrl, imdbUrl,
+  } = movieData;
```

Comment:
```suggestion
export const MovieCard: React.FC<Props> = (props) => {
  const { movieData } = props;
  const {
    title, description = '', imgUrl, imdbUrl,
  } = movieData;
```
Let's stick to this usage of props

Reply:
I've fixed that, but I have a question: is it correct to do that in such way:
```
const {
    title, description = '', imgUrl, imdbUrl,
  } = props.movieData;
```

Reply:
Better to write all those props on newline each